src/styles/section3.scss

```diff
@@ -0,0 +1,326 @@
+                padding-right: 0.375rem;
+              }
+        }
+
+        .product-description {
+            line-height: 1.5rem;
+                font-size: 0.875rem;
+                letter-spacing: 0;
+                font-weight: 400;
+                height: 0.875rem;
+                margin-top: 0.5rem;
+                color: #2e261c;
+        }
+    }
+
+        .product-rating {
+          line-height: 17px;
+          font-family: wotfard, sans-serif;
+
+          i {
+            color: #000;
+            font-size: 14px;
+          }
+
+          span {
+            font-size: 14px;
+          }
+    }
+
+    .product-price {
+        padding-top: 0.75rem;
+        text-transform: uppercase;
+        padding-bottom: 0.25rem;
+        border: none;
+        border-bottom: 2px solid #c65238;
+        text-align: left;
+        margin-top: 2rem;
+        background-color: #fff;
+    }
+    }
+  }
+}
+
+@media (max-width: 768px) {
+    #best-sellers-section {
+        padding-inline: 2rem;
+        margin-inline: 0;
+
+        .controls .swiper-scrollbar {
+            color: #fff;
+            width: 40%;
+            background: #fff;
+            display: flex;
+            margin-inline: 30%;
+          }
+    }
+}
+
+@media (max-width: 1500px) {
+  #best-sellers-section {
+    padding-inline: 4rem;
+    margin-inline: 0.5rem;
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    width: 100% !important;
+    height: 100%;
+
+    .swiper-container {
+        max-width: 68.875rem;
+        margin-inline: auto !important;
+        display: flex;
+        overflow: hidden;
+        width: 100%;
+    }
+
+    .section-header {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+      margin-bottom: 4rem;
+
+      .section-title {
+        font-family: ivymode, sans-serif;
+        line-height: 3rem;
+        letter-spacing: 1.5px;
+        font-weight: 400;
+        font-size: 3rem;
+        margin-bottom: 16px;
+      }
+
+      .header-link {
+        font-family: wotfard, sans-serif;
+        color: #2e261c;
+        text-decoration: none;
+        margin-top: 1rem;
+        line-height: 1rem;
+        padding-bottom: 0.25rem;
+        border-bottom: 2px solid #c65238;
+        letter-spacing: 2.5px;
+        font-weight: 500;
+        font-size: 0.875rem;
+        margin-bottom: 16px;
+      }
+    }
+
+    #responsive-swiper {
+        margin: 0 !important;
+    }
+
+    .swiper-button-next,
+  .swiper-button-prev {
+      display: flex;
+  }
+
+  .controls {
+      margin-top: 4rem;
+      max-width: 40rem;
+
+  .swiper-pagination {
+      font-size: 1rem;
+      line-height: 1.5rem;
+      display: block;
+  }
+}
+
+.controls .swiper-scrollbar {
+    color: #fff;
+    width: 40%;
+    background: #fff;
+    display: flex;
+    margin-inline: 30%;
+  }
+
+    .swiper {
+        width: 100% !important;
```

Comment:
@Ragnarrlothbrok Try to avoid using of `!important` CSS property. You will end up having a specificity issue.